tests/test_core.py

```diff
@@ -7330,7 +7330,7 @@ def test_embind_4(self):
     self.do_run(src, '107')
 
   def test_embind_5(self):
-    self.emcc_args += ['--bind']
+    self.emcc_args += ['--bind', '-s', 'EXIT_RUNTIME=1']
```

Comment:
enabling the exit runtime here tests that the leak is properly ignored with asan enabled instead of just lsan.

Reply:
So would this have failed before this change in asan or lsan mode?

Reply:
Without explicitly opting in to the exit runtime, Asan doesn’t check leaks at all. So this change just ensures that we check the leaks in asan mode. Without it, the test will still pass because no leaks will be detected, but for the wrong reasons.